circle.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
 
 test:
   override:
-    - stack test
+    - stack test --test-arguments "--skip \"returns a valid openapi\""
```

Comment:
I agree, this is fine to skip on CI. We run the tests locally all the time anyway and this one is unlikely to break and comes from a library that probably has its own test suite.